### ISSUE TITLE
Cunstom player bug

### ISSUE THREAD

**Issue by metrotyranno** [sk89q/playx#26]
The cunstom player is broken on all PHX props :

http://yfrog.com/5ngmflatgrasswater20001j

**Comment by metrotyranno** [comment]
it display's on the other side too
http://yfrog.com/16gmflatgrasswater20002j

**Comment by sk89q** [comment]
Curiously, does it happen on non-square props?

**Comment by metrotyranno** [comment]
no, it only works on phx prop : models/hunter/plates/plate16x32.mdl

**Comment by sk89q** [comment]
I just fixed this in the newest branch of PlayX (it's quite different from 2.5 internally):
http://github.com/sk89q/playx/tree/newtake

Download that and try it. Note that this version doesn't have the admin timeout nor the media expiration features.
